orion-sdk-java/src/main/java/org/hyperledger/orion/sdk/Iterator.java

```diff
@@ -0,0 +1,18 @@
+// Copyright IBM Corp. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package org.hyperledger.orion.sdk;
+
+/**
+ * Iterator implements methods to iterate over a set records.
+ * 
+ */
+public interface Iterator {
```

Comment:
Java already has an Iterator interface

Reply:
We'll address this in the next release in order to not break the API for our i4Q users at this point. See #7